src/diffusers/models/upsampling.py

```diff
@@ -19,9 +19,13 @@
 import torch.nn.functional as F
 
 from ..utils import deprecate
+from ..utils.import_utils import is_torch_version
 from .normalization import RMSNorm
 
 
+is_torch_less_than_2_1 = is_torch_version("<", "2.1")
```

Comment:
We can define this within the function body itself.

Reply:
sounds good!